frontend/src/pages/dashboard/ExperimentsTable/ExperimentsTable.tsx

```diff
@@ -126,10 +139,41 @@ export function ExperimentsTableSuspendable({
       showCheckboxColumn,
       {
         id: "actions",
-        accessor: () => <ContextActions actions={[]} />,
+        accessor: (e: Experiment) => (
+          <ContextActions
+            actions={[
+              <div
+                key="metadata"
+                className="contextActions__item"
+                onMouseDown={async () => {
+                  const metadata = await cacheMetadata(e.id);
+                  setModalDialog({
+                    kind: "show-experiment-metadata",
+                    description: metadata.fileMetadata.concat(metadata.description),
```

Comment:
I see that now description is shown last, i. e. after all other metadata. I think description is likely to be more important than `fileMetadata` and it's better to show it first, but I am not entirely sure so it's up to you to decide.